### DISCUSSION TITLE
Support node-sass@6 (which supports Node 16)

### DISCUSSION THREAD

**Discussion by adrienharnay** [vercel/next.js#28704] [Ideas]
### Describe the feature you'd like to request

Hello,

I see in the [package.json](https://github.com/vercel/next.js/blob/c1a97e557b7fad7a87907a56d10f018401154edc/packages/next/package.json#L120) that the latest major version of node-sass, version 6, is not supported. This version [unlocks support for Node 16](https://github.com/sass/node-sass/releases/tag/v6.0.0).

### Describe the solution you'd like

Are there blocking points to support the new major? If so, how may I help getting them resolved?

### Describe alternatives you've considered

I know the docs suggest using `sass` instead of `node-sass`, but am relying on `node-sass` plugins that wouldn't work with `sass`.

**Comment by timneutkens** [comment]
I think all that's needed is updating the peerDependencies based on that sass-loader only changed the peerdeps to be updated.

**Reply by adrienharnay** [reply]
Related PR: https://github.com/vercel/next.js/pull/28710

**Comment by adrienharnay** [comment]
Ok thank you for this insight! I will submit a PR :+1:
